tsconfig.json

```diff
@@ -20,7 +20,7 @@
 		"paths": {
 			"@/*": ["./src/*"]
 		},
-		"target": "ES2017"
+		"target": "ES2022"
```

Comment:
Why not 2023?

Reply:
oh shit we are in 2024 already! I thought we are in 2022